bliss.js

```diff
@@ -363,12 +363,14 @@ extend($, {
 			}
 		}
 
-		if (env.method !== "GET" && !env.headers["Content-type"] && !env.headers["Content-Type"]) {
+		if (env.method !== "GET" && !("Content-type" in env.headers) && !("Content-Type" in env.headers)) {
```

Comment:
Why is this checked twice?

Reply:
`xhr.setRequestHeader()` accepts both "Content-Type" and "Content-type" ("T" vs "t"), but not for example all-lowercase "content-type". So it's case-sensitive but has an alias it seems. I can't find this in any spec. Probably some historic reason.